charts/k8s-monitoring/templates/platform_specific/openshift/security-context-constraint.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and (eq .Values.cluster.platform "openshift") .Values.logs.enabled .Values.logs.pod_logs.enabled }}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: {{ include "grafana-agent.fullname" (index .Subcharts "grafana-agent-logs") }}
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities: null
+defaultAddCapabilities: null
+defaultAllowPrivilegeEscalation: false
+forbiddenSysctls:
+- '*'
+fsGroup:
+  type: RunAsAny
+groups: []
+priority: null
+requiredDropCapabilities:
+- CHOWN
+- DAC_OVERRIDE
+- FSETID
+- FOWNER
+- SETGID
+- SETUID
+- SETPCAP
+- NET_BIND_SERVICE
+- KILL
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+seccompProfiles:
+  - runtime/default
+supplementalGroups:
+  type: RunAsAny
+users: []
+volumes:
+- configMap
+- emptyDir
+- hostPath
```

Comment:
This is not present in the openshift logging scc, maybe we don't need this?

Reply:
I think it is there, right?
```
var DesiredSCCVolumes = []security.FSType{"configMap", "secret", "emptyDir", "projected"}

func NewSCC() *security.SecurityContextConstraints {
...
	scc.Volumes = DesiredSCCVolumes
```

We're adding `hostPath` so the agent can read logs.

Reply:
https://github.com/openshift/cluster-logging-operator/blob/master/internal/auth/securitycontextconstraint.go#L30C1-L39

Reply:
I was referencing the `hostPath` value. I'm wondering how openshift logging accomplishes the same without it.

Also don't see this as a blocker. The changes look good as is, I'm just checking if we can do with less permissions :